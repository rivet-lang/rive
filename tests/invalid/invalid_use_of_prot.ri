prot fn protected_fn() {}
