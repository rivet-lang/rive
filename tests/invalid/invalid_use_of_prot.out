tests/invalid/invalid_use_of_prot.ri:1:1: error: invalid use of `prot`
  1 | prot fn protected_fn() {}
    | ^
    = note: only valid with classes and structs
rivetc: error: could not compile module `invalid_use_of_prot`, aborting due to previous error